test/e2e/upgrade_test.go

```diff
@@ -206,3 +168,31 @@ func (tc *testContext) scaleWMCODeployment(desiredReplicas int32) error {
 
 	return err
 }
+
+// deployWindowsWorkloadAndTester tests if the Windows Webserver deployment is available.
+// This is achieved by creating a Job object that continuously curls the webserver every 5 seconds.
+// returns a tearDown func that must be executed to cleanup resources
+func (tc *testContext) deployWindowsWorkloadAndTester() (func(), error) {
+	// create a Windows Webserver deployment
+	deployment, err := tc.deployWindowsWebServer("win-webserver", nil)
+	if err != nil {
+		return nil, errors.Wrap(err, "error creating Windows Webserver deployment for upgrade test")
+	}
+	// create a clusterIP service which can be used to reach the Windows webserver
+	intermediarySVC, err := tc.createService(deployment.Name, v1.ServiceTypeClusterIP, *deployment.Spec.Selector)
+	if err != nil {
+		return nil, errors.Wrapf(err, "error creating service for deployment %s", deployment.Name)
+	}
+	// create a Job object that continuously curls the webserver every 5 seconds.
+	testerJob, err := tc.createLinuxJob(windowsWorkloadTesterJob, generateLinuxWorkloadTesterCommand(intermediarySVC.Spec.ClusterIP))
+	if err != nil {
+		return nil, errors.Wrapf(err, "error creating linux job %s", windowsWorkloadTesterJob)
```

Comment:
Same comment here.

Reply:
Fixed in https://github.com/openshift/windows-machine-config-operator/pull/674/commits/267e616e832d92bbae86788c2612b4224a78d5a8